pipeline/src/helpers/error.ts

```diff
@@ -0,0 +1,60 @@
+import { ErrorRequestHandler } from "express";
```

Comment:
Can remove this whole file (and the express dependency) as per https://github.com/wellcomecollection/content-api/pull/15/files#r1147485796